Sprint-3/quote-generator/style.css

```diff
@@ -1 +1,36 @@
 /** Write your CSS in here **/
+body{
```

Comment:
There a couple unnecessary lines here that I can see. Try play around with some more css and see which lines. I’m talking about :)

Reply:
You're right some CSS lines here are unnecessary. I have changed many lines in the body - 5 and 6 unnecessary. I deleted the file box from the main class and some from  #new-quote